rbac/server/api/errors.py

```diff
@@ -96,15 +98,16 @@ class ApiInternalError(ApiException):
 
 @ERRORS_BP.exception(ApiException)
 def api_json_error(request, exception):
+    if not request:
```

Comment:
Why did you user two different methods of handling this here? at line 101 and at line 110? in one you remove it.  and in the other you  keep and add a logger statement?

Reply:
Woops! Sorry, a couple of methods were throwing errors during tests when I removed request, so I decided to add request back as an argument but I missed this one